### ISSUE TITLE
Consider another OLED screen

### ISSUE THREAD

**Issue by Qix-** [oro-os/oro-link-x86#18]
These OLED screens are either faulty or their docs are wrong. Two tests in a row have garnered absolutely nothing - no backlighting, no response, nothing from any test, and turning on the power leads to the cable getting hot (still researching if that's normal).

**Comment by Qix-** [comment]
I honestly thing it's our best bet. It's in supply, it's high resolution, it's a good shape, and it's cheap. It's white, which fits Oro's branding, and has decent documentation (for the actual display chip - the OLED itself has rather poor documentation).

Going to give it another go, but keep this open for now.

**Comment by Qix-** [comment]
Just tried this with a Nucleo development board whereby the data signals are indeed the correct voltage. Nada. The ribbon is still getting insanely hot.

I think I'm going to make a breakout board for this and try to get to the bottom of whatever is causing this issue.

**Comment by Qix-** [comment]
Finally found the problem with OLEDs.

Made a breakout board for the OLED port and just got it in yesterday. Soldered it all together today, hooked up one of the units and did some basic "same net" testing (idk what the real term is).

Turns out, not much connects, but a few pins - 1, 24, 26, and 30 - did. On the existing schematic, 1 and 30 were NC, and 24 and 26 were two of the bits of the data bus, which made _absolutely no sense_.

But it did. If you flip the pins upside-down, they correspond to (existing pins) 2, 6, 8 and 31. If you check the datasheet for the OLED screen, those are all of the 12v pins exactly.

So I went back and checked the datasheet for the connector itself. Port numbering looks correct. Then I check the datasheet for the OLEDs and notice something a little annoying.

![image](https://github.com/oro-os/oro-link-x86/assets/885648/048c32e3-707c-4108-a4a8-489c68ce534f)

Yep. That's the _bottom_ of the connector. I suppose it makes sense, it's just highly confusing. The ports are indeed flipped, and the two datasheets do not agree with the port numbering. I suppose lesson learned.

Will wire up the breakout board to a development Nucleo board soon just to make sure _something_ is working but as it stands this has solved the elusive OLED screen problem.

Going to close.
